pkgs/development/python-modules/monai/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, numpy
+, pytorch
+}:
+
+buildPythonPackage rec {
+  pname = "monai";
+  version = "0.8.0";
+
+  disabled = pythonOlder "3.6";
+
+  format = "wheel";
+
+  # building from source with Ninja attempts to access network, so use PyPI wheel instead
+  src = fetchPypi {
+    inherit pname format;
+    version = "${version}-202111251823";
```

Comment:
This needs to be moved to version or automatic updates will do weird things.

Reply:
Makes sense, fixed.